agenta-web/src/components/Evaluations/SingleModelEvaluationCardView/index.tsx

```diff
@@ -0,0 +1,429 @@
+                            <EvaluationInputs
+                                evaluationScenario={scenario}
+                                onInputChange={onInputChange}
+                            />
+                        )}
+
+                        <div className={classes.toolBar}>
+                            <Tooltip title="Instructions">
+                                <QuestionCircleOutlined
+                                    onClick={showInstructions}
+                                    style={{color: token.colorPrimary}}
+                                />
+                            </Tooltip>
+                            {scenario.isPinned ? (
+                                <Tooltip title="Unpin">
+                                    <PushpinFilled
+                                        style={{color: token.colorErrorActive}}
+                                        onClick={() =>
+                                            updateEvaluationScenarioData(scenarioId, {
+                                                isPinned: false,
+                                            })
+                                        }
+                                    />
+                                </Tooltip>
+                            ) : (
+                                <Tooltip title="Pin">
+                                    <PushpinOutlined
+                                        style={{color: token.colorError}}
+                                        onClick={() =>
+                                            updateEvaluationScenarioData(scenarioId, {
+                                                isPinned: true,
+                                            })
+                                        }
+                                    />
+                                </Tooltip>
+                            )}
+                            <Tooltip title="Run (Enter ↵)">
+                                <PlayCircleOutlined
+                                    style={{color: token.colorSuccessActive}}
+                                    onClick={() => onRun(scenarioId)}
+                                />
+                            </Tooltip>
+                        </div>
+
+                        <EvaluationCard
+                            isChat={isChat}
+                            variants={variants}
+                            evaluationScenario={scenario}
+                        />
+                    </div>
+
+                    <div className={classes.sideBar}>
+                        <Typography.Title level={4}>Submit your feedback</Typography.Title>
+                        {scenario.outputs.some((item) => !!item.variant_output) && (
+                            <Space direction="vertical">
+                                <Typography.Text strong>Rate the response</Typography.Text>
+                                <Space>
+                                    <Input
```

Comment:
please use InputNumber component here:
https://ant.design/components/input-number with min=0 and max=100 props